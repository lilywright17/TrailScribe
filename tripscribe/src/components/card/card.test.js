import React from 'react';
import Card from './card';
import renderer from 'react-test-renderer';

test('renders card component correctly with country, city passed by props', () => {

    const tree = renderer.create(<Card country="Spain" city="Madrid" />).toJSON();
    expect(tree).toMatchSnapshot();
}
);

// test to see if image is rendered correctly
test('renders card component correctly with image passed by props', () => {
    const tree = renderer.create(<Card image="https://www.example.com/image.jpg" />).toJSON();
    expect(tree).toMatchSnapshot();
}
);