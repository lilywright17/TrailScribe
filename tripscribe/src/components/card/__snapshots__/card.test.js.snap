// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card component correctly with country, city passed by props 1`] = `
<div
  className="card"
  onClick={[Function]}
>
  <img
    alt="Madrid"
    className="card-img"
    width={200}
  />
  <h1>
    Spain
  </h1>
  <h2>
    Madrid
  </h2>
  <p>
    start date
     - 
    end date
  </p>
  <p>
    description of trip
    ...
  </p>
  <img
    alt="Edit button"
    className="edit-button"
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  />
</div>
`;

exports[`renders card component correctly with image passed by props 1`] = `
<div
  className="card"
  onClick={[Function]}
>
  <img
    alt="photo of city"
    className="card-img"
    width={200}
  />
  <h1>
    country
  </h1>
  <h2>
    town/city
  </h2>
  <p>
    start date
     - 
    end date
  </p>
  <p>
    description of trip
    ...
  </p>
  <img
    alt="Edit button"
    className="edit-button"
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  />
</div>
`;
